jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -162,6 +191,27 @@ public static boolean isCRIUSupportEnabled() {
 	private boolean trackMemory;
 	private Path envFile;
 
+	@SuppressWarnings("deprecation")
+	private static boolean loadNativeLibrary() {
```

Comment:
this should also be `synchronized` or two threads may race to load the library before the `nativeLoaded` boolean is set